pkgs/top-level/all-packages.nix

```diff
@@ -12026,8 +12026,7 @@ in
   cpplint = callPackage ../development/tools/analysis/cpplint { };
 
   ccls = callPackage ../development/tools/misc/ccls {
-    llvmPackages = llvmPackages_8;
-    stdenv = llvmPackages_8.stdenv;
```

Comment:
This was done to fix the build on macOS: https://github.com/NixOS/nixpkgs/pull/56673/files#diff-ab5748dc9567516fefba8344056b51ec1866adeace380f46e58a7af3d619ea22R8582
Did we update clang in the meantime?

Reply:
Yep it looks like it. Clang 7.1.0 is used by default for macOS's stdenv. I used [OSX-KVM](https://github.com/kholia/OSX-KVM) to test the build.